SignalServiceKit/src/Util/OWSLogger.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+import Foundation
+
+// Once we're on Swift4.2 we can mark this as inlineable
+// @inlinable
+public func owsFormatLogMessage(_ logString: String,
+                    file: String = #file,
+                    function: String = #function,
+                    line: Int = #line) -> String {
+    let filename = (file as NSString).lastPathComponent
+    // We format the filename & line number in a format compatible
+    // with XCode's "Open Quickly..." feature.
+    return "[\(filename):\(line) \(function)]: \(logString)"
+}
+
+/**
+ * A minimal DDLog wrapper for swift.
+ */
+open class Logger: NSObject {
+
+    open class func verbose(_ logString: @escaping @autoclosure () -> String,
+                            file: String = #file,
+                            function: String = #function,
+                            line: Int = #line) {
+        #if DEBUG
```

Comment:
My proposal is that this looks something like:
```
guard ddLogLevel <= verbose else {
  return
}
let message = owsFormatLogMessage(logString(), file: file, function: function, line: line)
OWSLogger.verbose(message)
```

Then remove the #if DEBUG and change the params of the OWSLogger to be a String, not a (()->String)

Also, can you rename this file to Logger.swift?

Reply:
Done, Done, Done, Done.